### PR TITLE
User menu

### PR DESCRIPTION
**Pull request by jeremiehelme**
**User Menu**
Changed the position of the buttons to the right
Fixed the font, size and colors for the different screens

user-menu

### CODE REVIEW

File: public/css/dashboard.css [hunk 1]
```diff
@@ -112,16 +112,23 @@ a {
...
-    background-color: var(--header-background-color);
+    padding-left: 0px;
     grid-area: user;
     display: flex;
     align-items: center;
-    flex-direction: row-reverse;
     text-align: right;
     font-size: 30px;
     color: var(--header-color);
+    position: absolute;
```
Comment (SamMousa) [inline public/css/dashboard.css]: Why is the user menu absolutely positioned?

File: public/css/dashboard.css [hunk 2]
```diff
@@ -131,16 +138,24 @@ a {
     border-radius: 50%;
 }
 .user-menu .name{
-    color: inherit;
+    color: #fefefe;
```
Comment (SamMousa) [inline public/css/dashboard.css]: Color codes should always go into variables.

**Comment by SamMousa** [comment]
Please add screenshots of the different screen sizes, each breakpoint should have a different screenshot

File: public/css/dashboard.css [hunk 3]
```diff
@@ -488,7 +497,7 @@ body {
                 var(--main-background-color) 100%)
     ;
         grid-template-rows: auto auto 1fr;
-        grid-template-columns: auto 1fr 1fr 1fr;
+        grid-template-columns: auto 1fr 1fr 0fr;
```
Comment (SamMousa) [inline public/css/dashboard.css]: Why have an empty column?

File: public/css/dashboard.css [hunk 4]
```diff
@@ -465,20 +477,17 @@ body {
         grid-row: span 2;
     }
 
+    body.no-title .user-menu {
```
Comment (SamMousa) [inline public/css/dashboard.css]: Does this work on smaller screens? I think it should not be fixed on smaller screens right?

File: public/css/dashboard.css (hunk 3 above)
Comment (jeremiehelme) [reply public/css/dashboard.css]: Oups, yes sorry, that's a mistake

File: public/css/dashboard.css (hunk 4 above)
Comment (jeremiehelme) [reply public/css/dashboard.css]: It is now fixed only for no-title screens above 800px width.
I don't find a way to display it above the map with the grid system

File: public/css/dashboard.css [hunk 5]
```diff
@@ -488,7 +500,7 @@ body {
                 var(--main-background-color) 100%)
     ;
         grid-template-rows: auto auto 1fr;
-        grid-template-columns: auto 1fr 1fr 1fr;
+        grid-template-columns: auto 1fr 1fr 200px;
```
Comment (SamMousa) [inline public/css/dashboard.css]: Why is the last column fixed width? We do not have a column on the right in our layout as far as I'm aware?

Comment (jeremiehelme) [reply public/css/dashboard.css]: Ok, I see what you mean. I changed the width of the user menu instead

**Comment by jeremiehelme** [comment]
@SamMousa The error in last review has been fixed, I think it can be merged

**Pull request merged**